### PR TITLE
Lesson 33 State

### PR DESCRIPTION
**Pull request by M-fkstv**

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  const setAppState = (value) => {
+    setState(value);
+  };
+
+  return (
+    <>
+      <Sprite />
+      <Main onSubmit={setAppState} />
+      {state.length > 0 ? (
```
Comment (almuly) [inline src/Components/App/App.js]: it's redaundant write ternary operator, whre on of argumnt boolean false. So you should use && instead. It's will be look like  {state.length > 0 && (
        <Section sectionName={"Available hotels"} data={state} />
      )}

File: src/Components/Form/Form.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  function handleChange(e) {
+    searchValue = e.target.value;
+  }
+  return (
+    <form id="form" className="form col-md-12" onSubmit={handleSubmit}>
+      <HotelsSearch onChange={handleChange} />
+      {/*<div className="form__city col-md-4">*/}
+      {/*<label className="form__city--label label" htmlFor="search">*/}
+      {/*  Your destination or hotel name*/}
+      {/*</label>*/}
```
Comment (almuly) [inline src/Components/Form/Form.js]: It's a bad style to left commented  code in your project

File: src/Components/Form/Form.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+      }
+      return result;
+    });
+    onSubmit([...result]);
+  }
+
+  let searchValue;
+  function handleChange(e) {
+    searchValue = e.target.value;
+  }
```
Comment (almuly) [inline src/Components/Form/Form.js:36]: Use useState hook instead
It easier to mange it and after learning more hooks, yo will escape extra re-renders
let [searchValue, setSearchValue]=useState(null) and etc.

File: src/Components/Form/Form.js (hunk 2 above)
Comment (M-fkstv) [reply src/Components/Form/Form.js]: Sorry for that

File: src/Components/App/App.js (hunk 1 above)
Comment (M-fkstv) [reply src/Components/App/App.js]: Done

File: src/Components/Form/Form.js (hunk 3 above)
Comment (M-fkstv) [reply src/Components/Form/Form.js:36]: Done

**Review by almuly — APPROVED** [review]

**Comment by almuly** [comment]
Well done!

**Pull request merged**